infra/github-runner/dev/locals.tf

```diff
@@ -0,0 +1,12 @@
+locals {
+  prefix    = "dx"
+  env_short = "d"
+  repo_name = "dx-playground"
+
+  tags = {
+    CostCenter  = "TS310 - PAGAMENTI & SERVIZI"
```

Comment:
```suggestion
    CostCenter  = "TS700 - ENGINEERING"
```

Reply:
@gunzip should we use that CostCenter for every resource?

Reply:
@gunzip I think CostCenter should be addressed to the right value almost everywhere (we're using the old TS310). Can you confirm TS is the right one?

Reply:
yes, we should use TS700 for every dx resource

Reply:
[This PR](https://github.com/pagopa/dx-playground/pull/21) is going to change the CostCenter accordingly.